com.microsoft.mrtk.graphicstools.unity/Editor/ShaderGUIs/StandardShaderGUI.cs

```diff
@@ -208,6 +209,7 @@ protected static class Styles
         protected MaterialProperty normalMap;
         protected MaterialProperty enableEmission;
         protected MaterialProperty emissiveColor;
+        protected MaterialProperty emissiveMap;

```

Comment:
Could you also copy the [emission map](https://github.com/TwoTailsGames/Unity-Built-in-Shaders/blob/master/DefaultResourcesExtra/Standard.shader#L32) property from the standard shader in the [AssignNewShaderToMaterial](https://github.com/microsoft/MixedReality-GraphicsTools-Unity/blob/385952c3e0c59b934f7846cb2b815977cb22dcda/com.microsoft.mrtk.graphicstools.unity/Editor/ShaderGUIs/StandardShaderGUI.cs#L495) function below? 

This way users will keep their settings when switching shaders. 😊